test/library/packages/TOML/test/TomlTest.compopts

```diff
@@ -0,0 +1,9 @@
+#
+# Since these tests modify the .good file as part of the test, we cannot add
+# compiler warnings for deprecated array operations to the .good file.
+# Instead, we suppress all compiler warnings (until TOML is rewritten to not
+# use "array-as-vec" operations).
+#
+# See: https://github.com/chapel-lang/chapel/issues/13233
```

Comment:
I would expect that TOML is just using 1-D arrays and so could easily switch to using `list` ?